package.json

```diff
@@ -10,21 +10,19 @@
     "chat",
     "bot"
   ],
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "http://github.com/hipchat/hubot-hipchat/raw/master/LICENSE"
-    }
-  ],
   "repository": {
     "type": "git",
     "url": "http://github.com/hipchat/hubot-hipchat.git"
   },
   "license": "MIT",
   "main": "./src/hipchat",
+  "engines": {
+    "node" : "~0.12.2"
+  },
   "dependencies": {
-    "node-xmpp": "~1.0.0-alpha2",
```

Comment:
@jzellner Pulling in the latest changes from upstream - it looks like they have an even more up-to-date version of this than the one you added not long ago.  Any issue with updating this?